public/app/features/alerting/unified/components/rules/deleted-rules/DeletedRulesPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { alertRuleApi } from '../../../api/alertRuleApi';
+import { GRAFANA_RULER_CONFIG } from '../../../api/featureDiscoveryApi';
+import { stringifyErrorLike } from '../../../utils/misc';
+import { withPageErrorBoundary } from '../../../withPageErrorBoundary';
+import { AlertingPageWrapper } from '../../AlertingPageWrapper';
+
+import { DeletedRules } from './DeletedRules';
+
+function DeletedrulesPage() {
+  const {
+    currentData = [],
+    isLoading,
+    error,
+  } = alertRuleApi.endpoints.getDeletedRules.useQuery({
+    rulerConfig: GRAFANA_RULER_CONFIG,
+    filter: {}, // todo: add filters, and limit?????
+  });
+  const values = Object.values(currentData);
+  const deletedRules = values.length > 0 ? values[0][0]?.rules : [];
```

Comment:
It feels like some of this manipulation could happen in the API endpoint instead? As `transformResponse`?

Reply:
good idea!